### PR TITLE
Added proper inhand sprites for the Lightfall

### PR DESCRIPTION
**Pull request by PigeonLord**
## Changelog
:cl:
imageadd: Added proper inhand sprites for the Lightfall 
/:cl:

Now it actually looks like you're holding a lightfall instead of three greenish pixels!

### CODE REVIEW

File: code/modules/projectiles/guns/energy/laser.dm [hunk 1]
```diff
@@ -19,6 +20,7 @@
 		WEAPON_NORMAL,
 		WEAPON_CHARGE
 	)
+	one_hand_penalty = 5
```
Comment (GrayRachnid) [inline code/modules/projectiles/guns/energy/laser.dm:23]: This isn't necessary, the gun cannot be used one handed.

**Comment by GrayRachnid** [comment]
Have you tested this in game? Can you show screenshots of all sides?

**Comment by PigeonLord** [comment]
lightfallproof
Changes tested and errors fixed

File: code/modules/projectiles/gun.dm [hunk 2]
```diff
@@ -107,7 +107,7 @@
 				slot_s_store_str = icon,
 			)
 		item_icons = item_icons_cache[type]
-	if(one_hand_penalty && (!wielded_item_state))//If the gun has a one handed penalty but no wielded item state then use this generic one.
+	if(one_hand_penalty || twohanded && (!wielded_item_state))//If the gun has a one hand penalty or is twohanded but no wielded item state then use this generic one.
```
Comment (mattroks101) [inline code/modules/projectiles/gun.dm:110]: Why did you change my comment to be grammatically incorrect?

Comment (PigeonLord) [reply code/modules/projectiles/gun.dm:110]: I fucked up and accidentally deleted the "one handed" part to write the twohanded thing. Sorry. Does it really matter that I fix it? I kinda don't want to.

Comment (mattroks101) [reply code/modules/projectiles/gun.dm:110]: Yeah it matters, you made the comment less readable by butchering the grammar that was already correct.

Comment (PigeonLord) [reply code/modules/projectiles/gun.dm:110]: Fine, fuck me

Comment (PigeonLord) [reply code/modules/projectiles/gun.dm]: Fixed the typo, and also fixed a typo YOU made in the original comment, you fucking weenie.
